signs-work-order-map/components/SignPopup.tsx

```diff
@@ -0,0 +1,40 @@
+import { Popup } from "react-map-gl/mapbox";
+import { Sign } from "@/types/map";
+
+interface SignPopupProps {
+  popupInfo: Sign;
+  setPopupInfo: React.Dispatch<React.SetStateAction<Sign | null>>;
+}
+
+export default function SignPopup({ popupInfo, setPopupInfo }: SignPopupProps) {
+  // prod has the font as bold, do we keep?
+  return (
+    <Popup
+      anchor="top"
+      longitude={popupInfo.lng}
+      latitude={popupInfo.lat}
+      onClose={() => setPopupInfo(null)}
+      offset={[0, -20]}
+    >
+      <ul className="list-unstyled m-0">
+        {!popupInfo.isLocationDetailPage && (
+          <li>
+            <a
+            // TODO: update to prod url when testing is completed
+              href={`https://atd.knack.com/test-30-may-2024-signs-and-markings-operations#work-order-signs/view-work-orders-details-sign/${popupInfo?.workOrderId}/view-work-order-signs-location-details/${
+                popupInfo.id
+              }`}
+              // ensure it doesn't open in the iframe
+              target="_top"
+            >
+              Location Detail Page
+            </a>
+          </li>
+        )}
+        <li>Spatial ID: {popupInfo.spatialId}</li>
+        <li>Latitude: {popupInfo.lat}</li>
+        <li>Longitude: {popupInfo.lng}</li>
+      </ul>
```

Comment:
I know it's out of scope, but fwiw these pop-ups might have some usability issues due to how small they are.

<img width="715" alt="Screenshot 2025-06-18 at 2 38 08 PM" src="https://github.com/user-attachments/assets/cb4f1435-5546-4507-95e0-8d4c294253e0" />

If it's helpful, on https://data.mobility.austin.gov/ we use modal for popups on smaller displays

<img width="1512" alt="Screenshot 2025-06-18 at 3 02 32 PM" src="https://github.com/user-attachments/assets/6944fab7-8e9e-418e-b6af-5f8ca851b0e5" />

<img width="569" alt="Screenshot 2025-06-18 at 3 02 27 PM" src="https://github.com/user-attachments/assets/dd7da44e-1a75-4335-910c-064939744f9d" />